### PR TITLE
Add WordPress version for automatic update checks

### LINKED ISSUE #1054: Automate updates to WordPress.
The updates to WordPress could be automated via a GitHub action running on a schedule.

The process could be something along the following lines:

* Query `http://api.wordpress.org/core/version-check/1.7/` (using HTTP ensures only the latest release is shown)
* Regex to figure out the latest tag
* If the tag does not match the current version of WP in Chassis, create a PR to run an update

As new releases are tagged prior to the official release, we're unable to use the GitHub API to query tags on https://github.com/wordpress/wordpress as it's theoretically possible a release gets abandoned post tagging if a serious issue arises.

### PR DESCRIPTION
**Pull request by BronsonQuick**
Fixes #1054

### CODE REVIEW

File: .github/workflows/worpdress-updates.yml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    # Check for WordPress updates every hour.
+    - cron: 0 * * * *
+jobs:
+  maybe-update-wordpress:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Compare Versions
+        id: compare-versions
+        uses: actions/checkout@v4
+      # Install jq to parse JSON.
```
Comment (peterwilsoncc) [inline .github/workflows/worpdress-updates.yml:14]:
```suggestion
      - name: Install PHP
        uses: shivammathur/setup-php@v2
      # Install jq to parse JSON.
```

File: .github/workflows/worpdress-updates.yml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+      - run: sudo apt-get install jq
+      - run: echo "CURRENT_WP_VERSION=$(curl -s https://api.wordpress.org/core/version-check/1.7/ | jq -r '.offers[0].current')" >> "$GITHUB_ENV"
+      - run: echo "Current WordPress version is ${{ env.CURRENT_WP_VERSION }}"
+      # Install yq to parse YAML.
+      - run: wget https://github.com/mikefarah/yq/releases/download/v4.40.5/yq_linux_amd64.tar.gz -O - | tar xz && sudo mv yq_linux_amd64 /usr/bin/yq
+      - run: echo "CHASSIS_WP_VERSION=$(yq .wp_version config.yaml)" >> "$GITHUB_ENV"
+      - run: echo "Chassis WordPress version is ${{ env.CHASSIS_WP_VERSION }}"
```
Comment (peterwilsoncc) [inline .github/workflows/worpdress-updates.yml:21]:
```suggestion
      # Get installed version of WP.
      - run: echo "CHASSIS_WP_VERSION=$(php -r 'require "wp/wp-includes/version.php"; echo $wp_version;')" >> "$GITHUB_ENV"
      - run: echo "Chassis WordPress version is ${{ env.CHASSIS_WP_VERSION }}"
```

**Review by peterwilsoncc — COMMENTED** [review]
Added a few notes, I'm going to test this on my branch though.

**Comment by BronsonQuick** [comment]
Closed in favour of: https://github.com/Chassis/Chassis/pull/1060

**Pull request closed without merging**